ui/js/controllers/logviewer.js

```diff
@@ -209,7 +209,8 @@ logViewerApp.controller('LogviewerCtrl', [
             }).catch((error) => {
                 $scope.loading = false;
                 $scope.jobExists = false;
-                thNotify.send(`${error}`, 'danger', { sticky: true });
+                $scope.jobError = error.toString();
```

Comment:
This leaves the logviewer with a blank page, but error message at the top stating that the job wasn't found.  And then it doesn't need ``notify`` at all.